packages/react-core/src/demos/TableColumnManagementDemo.md

```diff
@@ -0,0 +1,608 @@
+          },
+          {
+            title: (
+              <React.Fragment><a href="#">Action link</a></React.Fragment>
+            ),
+            props: { column: '' }
+          }
+        ]
+      },
+      {
+        cells: [
+          {
+            title: (
+              <React.Fragment>
+                <div>Node 5</div>
+                <a href="#">siemur/test-space</a>
+              </React.Fragment>
+            ),
+            props: { column: 'Repositories' }
+          },
+          {
+            title: (
+              <React.Fragment>
+                <CodeBranchIcon key="icon" /> 34
+              </React.Fragment>
+            ),
+            props: { column: 'Branches' }
+          },
+          {
+            title: (
+              <React.Fragment><CodeIcon key="icon" /> 21</React.Fragment>
+            ),
+            props: { column: 'Pull requests' }
+          },
+          {
+            title: (
+              <React.Fragment><CubeIcon key="icon" /> 26</React.Fragment>
+            ),
+            props: { column: 'Workspaces' }
+          },
+          {
+            title: '2 days ago',
+            props: { column: 'Last commit' }
+          },
+          {
+            title: (
+              <React.Fragment><a href="#">Action link</a></React.Fragment>
+            ),
+            props: { column: '' }
+          }
+        ]
+      },
+    ];
+    this.state = {
+      filters: [],
+      filteredColumns: [],
+      filteredRows: [],
+      columns: this.defaultColumns,
+      rows: this.defaultRows,
+      canSelectAll: true,
+      isModalOpen: false,
+      check1: true,
+      check2: true,
+      check3: true,
+      check4: true,
+      check5: true
+    };
+    this.onSelect = this.onSelect.bind(this);
+    this.toggleSelect = this.toggleSelect.bind(this);
+    this.renderModal = this.renderModal.bind(this);
+    this.matchLongCheckboxNameToShort = (name) => {
+      switch (name) {
+        case 'table-column-management-check-action-check1':
+          return 'check1';
+        case 'table-column-management-check-action-check2':
+          return 'check2';
+        case 'table-column-management-check-action-check3':
+          return 'check3';
+        case 'table-column-management-check-action-check4':
+          return 'check4';
+        case 'table-column-management-check-action-check5':
+          return 'check5';
+      }
+    };
+    this.matchCheckboxNameToColumn = (name) => {
+      switch (name) {
+        case 'table-column-management-check-action-check1':
+          return 'Repositories';
+        case 'table-column-management-check-action-check2':
+          return 'Branches';
+        case 'table-column-management-check-action-check3':
+          return 'Pull requests';
+        case 'table-column-management-check-action-check4':
+          return 'Workspaces';
+        case 'table-column-management-check-action-check5':
+          return 'Last commit';
+      }
+    };
```

Comment:
I find the name conversions confusing. In Cost Management, we just provide the names we intend to use without mappings.

Can we not provide the table checkbox with the name "check1", "check2", etc., for the purpose of this example? (My first preference)  Or, change the state names to use the matching name?

Reply:
I was copying the id scheme Core was using. Just got @mcoker's blessing to diverge.